src/libsystemd-network/test-dhcp-server.c

```diff
@@ -62,7 +62,10 @@ static int test_basic(bool bind_to_interface) {
         test_pool(&address_lo, 1, 0);
 
         r = sd_dhcp_server_start(server);
-        if (r == -EPERM)
+
```

Comment:
Could you drop the empty line? We usually write error handling soon after the function call.